### PR TITLE
Feat - WIP-325 - Add who answered a question

### PR DESCRIPTION
**Pull request by timgivois**
#### What does this PR do?
- Displays the name of the person that answered a question if the name exists in the DB.

#### Where should the reviewer start?
- In src/components/LisQuestions.js check on line 215 renderAnswerRow.

#### How should this be manually tested?
1. Create an answer to a question and see in Answered Questions tabs if the answer has name.

#### Any background context you want to provide?
Depends on https://github.com/wizeline/wize-q-back/pull/50

#### What are the relevant tickets?
https://wizeline.atlassian.net/browse/WIP-325

### CODE REVIEW

File: src/components/ListQuestions.js [hunk 1]
```diff
@@ -135,6 +135,13 @@ class List extends Component {
     return username;
   }
 
+  displayAnsweredBy = (username) => {
+    if (username) {
+      return `Answered by ${username}`;
```
Comment (isa-bel) [inline src/components/ListQuestions.js]: Maybe you can use the ternary operator here?

File: src/styles/QuestionsAnswered.css [hunk 2]
```diff
@@ -1,7 +1,12 @@
 .answer {
```
Comment (isa-bel) [inline src/styles/QuestionsAnswered.css]: Can you add a main class `QuestionsAnswered` to the component and prefix all the rules with it? As in [ListQuestions.css](https://github.com/wizeline/wize-q-front/blob/master/src/styles/ListQuestions.css).

File: src/components/ListQuestions.js [hunk 3]
```diff
@@ -208,9 +213,15 @@ class List extends Component {
   renderAnswerRow = question => (
     <tr key={`a_${question.question_id}`}>
       <td className="col-md-1">&nbsp;</td>
-      <td className="col-md-7 col-xs-7">
-        {question.answer}
+      <td className="col-md-7 col-xs-7 QuestionsAnswered">
```
Comment (isa-bel) [inline src/components/ListQuestions.js]: I would put the main class first, so its clearer. Up to you.

**Comment by isa-bel** [comment]
Please update the PR title to fit to this template: https://github.com/wizeline/wize-docs/blob/master/development/git-contributing-guidelines.md#title

File: src/components/ListQuestions.js (hunk 3 above)
Comment (timgivois) [inline src/components/ListQuestions.js]: resolved

File: src/styles/QuestionsAnswered.css (hunk 2 above)
Comment (timgivois) [inline src/styles/QuestionsAnswered.css]: resolved

**Pull request merged**